search-service/src/main/kotlin/com/egm/stellio/search/authorization/AuthorizationUtils.kt

```diff
@@ -0,0 +1,15 @@
+package com.egm.stellio.search.authorization
+
+import com.egm.stellio.shared.util.AuthContextModel.COMPOUND_AUTHZ_CONTEXT
+import com.egm.stellio.shared.util.AuthContextModel.NGSILD_AUTHORIZATION_CONTEXT
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_CORE_CONTEXT
+
+fun List<String>.addAuthzContextIfNeeded(): List<String> =
```

Comment:
In authz endpoints, we consider the "default context" to be NGSI-LD Core Context + Authz context. Here, you are replacing the NGSI-LD Core Context by the Authz context.

Reply:
Ok for POST but when we do GET we don't have a list but a single string for context.

Reply:
yes, but you have a compound context for authz

Reply:
Maybe I could put these two functions in `AuthUtils` ?

Reply:
yes